src/library_glut.js

```diff
@@ -310,22 +336,27 @@ var LibraryGLUT = {
     GLUT.initTime = Date.now();
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
+    if (isTouchDevice) {
+      // Historically handling of touch events was made in completely wrong way.
```

Comment:
Yeah, fair point. We can just remove this comment, then? (it is already in the commit message)